boards/cuav/nora/src/board_config.h

```diff
@@ -199,7 +199,7 @@
 
 #define BOARD_HAS_PWM  DIRECT_PWM_OUTPUT_CHANNELS
 
-#define BOARD_DSHOT_MOTOR_ASSIGNMENT {3, 2, 1, 0, 4};
+#define BOARD_DSHOT_MOTOR_ASSIGNMENT {0, 1, 2, 3, 4, 5, 6, 7};
```

Comment:
You can remove it completely if it's the identity.

Reply:
That's better, thanks.